packages/cypress/README.md

```diff
@@ -0,0 +1,39 @@
+# @commercetools-frontend/cypress
+
+<p align="center">
+  <a href="https://www.npmjs.com/package/@commercetools-frontend/cypress"><img src="https://badgen.net/npm/v/@commercetools-frontend/cypress" alt="Latest release (latest dist-tag)" /></a> <a href="https://www.npmjs.com/package/@commercetools-frontend/cypress"><img src="https://badgen.net/npm/v/@commercetools-frontend/cypress/next" alt="Latest release (next dist-tag)" /></a> <a href="https://bundlephobia.com/result?p=@commercetools-frontend/cypress"><img src="https://badgen.net/bundlephobia/minzip/@commercetools-frontend/cypress" alt="Minified + GZipped size" /></a> <a href="https://github.com/commercetools/merchant-center-application-kit/blob/master/LICENSE"><img src="https://badgen.net/github/license/commercetools/merchant-center-application-kit" alt="GitHub license" /></a>
+</p>
+
+Cypress commands and utilities for Custom Applications.
+
+## Install
+
+```bash
+$ npm install --save @commercetools-frontend/cypress
+```
+
+## Usage
+
+This package extends Cypress' `cy` command.
+
+Add this line to your project's `cypress/plugins/index.js`:
+
+```javascript
+const customApplications = require('@commercetools-frontend/cypress/task');
+
+module.exports = (on, config) => {
```

Comment:
Not sure it applies but I still would recommend to install like this:

```js
customApplicationsPlugin.install(on, config);
```

This gives more freedom in the future. You can pass config options potentially and have access to the whole "on".

Reply:
I double checked this. We're exporting a `task`, which already implies that in the `cypress/plugins/index.js` you have to use `on('task', {...})`.

It does not make much sense to pass `on` to the task itself.

If we were to implement a plugin, then I guess it makes sense to do as you suggested.